samples/extensions/multipleroutes/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "manifest_version": 2,
+  "name": "WebRTC extension to disable multiple routes",
```

Comment:
"Extension to disable the option for multiple routes in WebRTC" might be a little better.